owncloudDomain/src/main/java/com/owncloud/android/domain/capabilities/model/OCCapability.kt

```diff
@@ -60,15 +61,25 @@ data class OCCapability(
         return filesSharingUserProfilePicture.isTrue || filesSharingUserProfilePicture.isUnknown
     }
 
-    fun isOpenInWebAllowed(): Boolean = filesOcisProviders?.openWebUrl?.isNotBlank() ?: false
+    fun isOpenInWebAllowed(): Boolean = filesAppProviders?.openWebUrlAppProviders?.isNotBlank() ?: false
 
-    data class OcisProvider(
-        val enabled: Boolean,
-        val version: String,
-        val appsUrl: String?,
-        val openUrl: String?,
-        val openWebUrl: String?,
-        val newUrl: String?,
+    fun isSpacesAllowed(): Boolean = spaces?.enabledSpaces == true
+
+    fun isSpacesProjectsAllowed(): Boolean = spaces?.projectsSpaces == true
+
+    data class AppProviders(
+        val enabledAppProviders: Boolean,
+        val versionAppProviders: String,
+        val appsUrlAppProviders: String?,
+        val openUrlAppProviders: String?,
+        val openWebUrlAppProviders: String?,
+        val newUrlAppProviders: String?,
```

Comment:
Ok, in the domain classes I changed it, and for the database I set a prefix to the @Embedded tag so that we can distinguish among variables with the same name 👍